src/watcher.js

```diff
@@ -333,6 +343,33 @@ class Watcher extends SafeEventEmitter {
         this.on('load.vod', () => observe(vodChatWatcher, $('.qa-vod-chat')[0]));
     }
 
+    vodRecommendationObserver() {
+        vodRecommendationWatcher = new window.MutationObserver(mutations => {
```

Comment:
rather than watching for vod recommendations, why not hook into the player events themselves and watch for the video end event and prevent the next video from being played?

Reply:
would this be the appropriate event to listen to `Twitch.Player.ENDED` ?